ci/infra/libvirt/lb-instances.tf

```diff
@@ -94,12 +127,15 @@ resource "libvirt_domain" "lb" {
   }
 
   disk {
-    volume_id = libvirt_volume.lb.id
+    volume_id = element(
+      libvirt_volume.lb.*.id,
+      count.index,
+    )
   }
 
   network_interface {
     network_id     = libvirt_network.network.id
-    hostname       = "${var.stack_name}-lb"
+    hostname       = "${var.stack_name}-lb-${count.index}"
```

Comment:
This will break `testrunner`'s logic for retrieving the lb's ip address:

https://github.com/SUSE/skuba/blob/f6fd30f300904f06262e3bcfc8d41f6b3b43c95e/ci/infra/testrunner/platforms/terraform.py#L77-L82

Reply:
How many LB do we need? I think 1 would be good enough... if that's the case I can just change the PR to allow the creation of 1 or 0 LB.

Reply:
I'm not quite it would work with '0' LBs. Need to double-check.

Reply:
No, it won't work. The existence of at least one LB was an assumption. 

https://github.com/SUSE/skuba/blob/f6fd30f300904f06262e3bcfc8d41f6b3b43c95e/ci/infra/testrunner/skuba/skuba.py#L51-L65

Reply:
The default value is still 1. I can revert back the change partially to ensure the name of the LB doesn't change.

It's still useful to allow the creation of 0 LB when doing small cluster deployments.

Reply:
Agreed on the 0 LB.

Reply:
I've updated the code to reflect that.